README.md

```diff
@@ -30,15 +35,14 @@ public class User : IEntity
 }
 ```
 
-Для запросов можно использовать любой тип для первичного ключа. Если использовать интерфейс *IEntity* для модели, в которых первичный ключ определен как *long*, то часть запросов будет проще, например, вставка всех полей, обновление всех полей, фильтр по ИД.
-
-Библиотека будет воспринимать все публичные свойства с присутствующими методами *get* и *set* как колонки таблицы. Их наименования будут соответствовать названиям колонок (для PostgreSQL используется форматирование SnakeCase).
+For queries, any type can be used for the primary key. If the *IEntity* interface is used for models, where the primary key is defined as *long*, some queries will be easier, for example, inserting all fields, updating all fields, filtering by ID.
 
+The library will treat all public properties with corresponding *get* and *set* methods as table columns. Their names will correspond to the column names (SnakeCase formatting is used for PostgreSQL).
 ## Select
 
-### Запрос сущности из таблицы
+### Querying an entity from a table.
 
-Для запросов SELECT нужно создать **SelectQuery** объект. Вот пример для сущности *Company*:
+To query an entity from a table, you need to create a **SelectQuery** object. Here is an example for the *Company* entity:
```

Comment:
To create SELECT query you will need to...

Reply:
I think that a proper word order in this particular case would be:  "You need to create a **SelectQuery** object to create a SELECT query."